src/client/src/main.tsx

```diff
@@ -29,19 +41,18 @@ const appBootstrapper = () => {
 
   const autobahn = new AutobahnConnectionProxy(url, realm, +port)
 
-  const openFin = new OpenFin()
-
   const applicationDependencies = createApplicationServices(user, autobahn, openFin)
 
-  const isRunningInFinsemble = window.FSBL
-
   const store = configureStore(applicationDependencies)
   window.store = store
+
   ReactDOM.render(
     <Provider store={store}>
-      <OpenFinProvider openFin={openFin} isRunningInFinsemble={isRunningInFinsemble}>
-        <ShellContainer />
-      </OpenFinProvider>
+      <EnvironmentProvider value={environmentContext}>
+        <OpenFinProvider openFin={openFin}>
```

Comment:
We should use a ternary to do this rather then adding the wrapper when we do not need it.